src/test/regress/sql/distributed_functions.sql

```diff
@@ -132,18 +166,29 @@ FROM pg_dist_partition, citus.pg_dist_object as objects
 WHERE pg_dist_partition.logicalrelid = 'replicated_table_func_test_4'::regclass AND 
 	  objects.objid = 'add_with_param_names(int, int)'::regprocedure;
 
--- if not paremeters are supplied, we'd see that function doesn't have
--- distribution_argument_index and colocationid
-SELECT create_distributed_function('add_mixed_param_names(int, int)');
-SELECT distribution_argument_index is NULL, colocationid is NULL from citus.pg_dist_object
-WHERE objid = 'add_mixed_param_names(int, int)'::regprocedure;
+-- clear objects
+SELECT stop_metadata_sync_to_node(nodename,nodeport) FROM pg_dist_node WHERE isactive AND noderole = 'primary';
 
--- also show that we can use the function
-SELECT * FROM run_command_on_workers('SELECT function_tests.add_mixed_param_names(2,3);') ORDER BY 1,2;
+SET client_min_messages TO error; -- suppress cascading objects dropping
+DROP SCHEMA function_tests CASCADE;
 
--- clear objects
+-- This is hacky, but we should clean-up the resources as below
+
```

Comment:
Why wasn't the DROP SCHEMA sufficient?

Reply:
Discussed offline. It does not actually remove the metadata, so leaves a lot of stuff for other tests.